gradebookng/tool/src/java/org/sakaiproject/gradebookng/framework/GradebookNgEntityProducer.java

```diff
@@ -129,12 +136,12 @@ public String archive(final String siteId, final Document doc, final Stack<Eleme
 		// <GradebookConfig>
 		Element gradebookConfigEl = doc.createElement("GradebookConfig");
 
-		Gradebook gradebook =  this.gradingService.getGradebook(siteId);
+		Gradebook gradebook =  this.gradingService.getGradebook(siteId, siteId);
```

Comment:
This is strange to me
```
		Gradebook gradebook =  this.gradingService.getGradebook(siteId, siteId);
```

Reply:
This is just for retrocompatibility and an exception case. It is called when archiving the tool and that functionality won't be covered for groups. I guess we could add a one parameter method just for that case but it feels redundant and we preferred it like this.